### PR TITLE
Fix: Chop git suffix from `LibLLVM::VERSION`

### PR DESCRIPTION
**Pull request by HOMODELUNA**

### CODE REVIEW

**Review by straight-shoota — APPROVED** [review]

File: src/llvm/lib_llvm.cr [hunk 1]
```diff
@@ -10,7 +10,7 @@ end
   {% end %}
   @[Link(ldflags: {{"`#{LibLLVM::LLVM_CONFIG} --libs --system-libs --ldflags#{" --link-static".id if flag?(:static)}#{" 2> /dev/null".id unless flag?(:win32)}`"}})]
   lib LibLLVM
-    VERSION = {{`#{LibLLVM::LLVM_CONFIG} --version`.chomp.stringify}}
+    VERSION = {{`#{LibLLVM::LLVM_CONFIG} --version`.chomp.stringify.gsub(/git/,"")}}
```
Comment (straight-shoota) [inline src/llvm/lib_llvm.cr:13]: For future reviewers: Note there is no equivalent to `String#rchop` for `StringLiteral` which might've been nice for this.

Comment (Sija) [inline src/llvm/lib_llvm.cr:13]: Could be tweaked a bit:

```suggestion
    VERSION = {{ `#{LibLLVM::LLVM_CONFIG} --version`.chomp.stringify.sub(/git$/, "") }}
```

Comment (straight-shoota) [reply src/llvm/lib_llvm.cr:13]: Hm, actually there's no need for a regex:
```suggestion
    VERSION = {{`#{LibLLVM::LLVM_CONFIG} --version`.chomp.stringify.sub("git","")}}
```

Comment (HertzDevil) [reply src/llvm/lib_llvm.cr:13]: There is no `StringLiteral#sub`, either

Comment (Sija) [reply src/llvm/lib_llvm.cr:13]: @straight-shoota It is, if you want to remove the `git` from the **end** of the string.

Comment (HertzDevil) [reply src/llvm/lib_llvm.cr:13]: That wouldn't work either, the macro `StringLiteral#gsub` requires a `RegexLiteral` first argument

Comment (HertzDevil) [reply src/llvm/lib_llvm.cr:13]: And of course, moving the `.gsub` or `.sub` outside `{{ ... }}` isn't an option as `LibLLVM::VERSION`'s initializer must be a literal

Comment (straight-shoota) [reply src/llvm/lib_llvm.cr:13]: Okay, so we leave everything as it is. Sorry for all the confusion 🙇

**Review by straight-shoota — APPROVED** [review]

File: src/llvm/lib_llvm.cr (hunk 1 above)
Comment (Sija) [reply src/llvm/lib_llvm.cr:13]: IMO `$` should be added, still.

**Comment by HertzDevil** [comment]
This still needs formatting

**Comment by beta-ziliani** [comment]
ah good catch

**Pull request merged**